chia/wallet/puzzles/did_innerpuz.clvm

```diff
@@ -132,19 +133,19 @@
   (if mode
     (if (= mode 1)
       ; mode one - create messages and recreate singleton
-      (c (list CREATE_COIN new_inner_puzhash amount) (c (list AGG_SIG_ME MY_PUBKEY (sha256tree1 (list new_inner_puzhash amount message))) (create_messages message)))
+      (c (list CREATE_COIN new_inner_puzhash new_amount) (c (list AGG_SIG_ME MY_PUBKEY (sha256tree1 (list new_inner_puzhash new_amount message))) (create_messages message)))
     ; mode two - recovery
     ; check that recovery list is not empty
       (if recovery_list_reveal
```

Comment:
You can use the `and` macro or the `all` keyword here instead of nested `if` statements.